reports/Indiana.md

```diff
@@ -44,14 +44,14 @@ Police confiscate medical supplies from a medical station at Monument Circle.
 
 * https://www.facebook.com/AlexandraIndy/videos/10219444419838711
 
-### Police beating, shoving, and shooting two women | June 1st
+### Police beat woman with batons and shoot pepper balls, shove another to the ground | June 1st
 
-One woman struggled against the officer restraining her, so she and her friend were beaten with batons, shot with rubber bullets, and shoved to the ground.
+A woman is seen being restrained by a police officer, who at one point moves his hand near her chest (it is unclear for what purpose, possibly to get a better hold, but the tweet alleges that it is evidence of sexual assault). She immediately struggles with the officer and breaks free, after which she is repeatedly beaten on the thighs with batons and shot with what appear to be pepper balls by officers shouting "Get on the ground!" while she is standing still. Another woman walks towards the officers with her hands out, seeming to plead for restraint but not directly interfering. An officer yells at her to "Back up!" and then violently shoves her to the ground.
```

Comment:
Happy for you to merge as-is, but in my opinion I don't think it is useful to include the allegation of sexual assault - though I'm not denying that may be the truth of what happened or that woman's experience.

I know the tweet alleges it, but the person writing that doesn't seem to have any more information than we do as they're aggregating videos from multiple sources and do not appear to have been present at the incident themself.

```suggestion
A woman is seen being restrained by a police officer who stands behind her with his arms around her torso. She struggles with the officer and breaks free, after which she is repeatedly beaten on the thighs with batons and shot with what appear to be pepper balls by officers shouting "Get on the ground!" while she is standing still. Another woman walks towards the officers with her hands out, seeming to plead for restraint but not directly interfering. An officer yells at her to "Back up!" and then violently shoves her to the ground.
```